docs/azure_data_studio/README_FOR_MARKETPLACE.md

```diff
@@ -101,3 +101,56 @@ For more information see the [Code of Conduct FAQ][conduct-FAQ] or contact [open
 [conduct-FAQ]: http://opensource.microsoft.com/codeofconduct/faq/
 [conduct-email]: mailto:opencode@microsoft.com
 [conduct-md]: https://github.com/PowerShell/vscode-powershell/blob/master/CODE_OF_CONDUCT.md
+
+## SQL PowerShell Examples
+In order to use these examples (below), you need to install the SqlServer module from the [PowerShell Gallery](https://www.powershellgallery.com/packages/SqlServer).
+
+<pre><code>Install-Module -Name SqlServer -AllowPrerelease</code></pre>
```

Comment:
Is there a reason we are recommending a prerelease version of this module rather than the last stable version?

Reply:
Ah, great catch.  I pulled that in from my recent post.  The latest Prerelease includes Invoke-Sqlcmd working on PSCore.  Since Azure Data Studio is cross-platform, having the x-plat Invoke-Sqlcmd is very useful and relevant to those customers.  We can either pull the " -AllowPrerelease" portion now, or we can wait until the final version of the module ships, and pull it then.  

Although, ironically, if PSCore still hasn't shipped then the customer would have to download that preview as well.  (We could include a link to the post on the [PowerShell blog](https://devblogs.microsoft.com/powershell/invoke-sqlcmd-is-now-available-supporting-cross-platform/) as well, to help the x-plat customers)  

In my mind, there is literally no other SQL PowerShell cmdlet as important as Invoke-Sqlcmd.  I can't ever see broadly suggesting customers use a -AllowPrerelease version after this one ships.

I'm fine with leaving " -AllowPrerelease" until the ned of the month, or pulling it now.  Up to you.

Reply:
I'm fine with `-AllowPrerelease` but please add a comment as to why we're installing the prerelease one.